### PR TITLE
Build two versions of new homepage

### LINKED ISSUE #254: Build two versions of newsletter sign-ups and setup AB test
- either build 1 page with js to show either version or 2 pages
- setup goal in GA
- setup test in Google optomise
- add Google optomise GTM tracker to u.c pages

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Build new versions of homepage

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Check homepage matches [this design](https://user-images.githubusercontent.com/36884067/37915540-a9a2bb12-3111-11e8-82e2-01683913baf1.jpg)
- View the site locally in your web browser at: [http://0.0.0.0:8023/home](http://0.0.0.0:8023/home)
- Check homepage matches [this design](https://user-images.githubusercontent.com/36884067/37915538-a978fd72-3111-11e8-8d07-f0db8580e101.jpg)

## Issue / Card

Fixes #254

### CODE REVIEW

**Comment by steverydz** [comment]
@anasereijo This is ready for a design review.

You can see the two designs at:

http://insights.ubuntu.com-pr-292.run.demo.haus/
http://insights.ubuntu.com-pr-292.run.demo.haus/home

File: static/sass/_pattern_card.scss [hunk 1]
```diff
@@ -55,7 +55,29 @@
     }
   }
 
+  .p-card--form {
```
Comment (anthonydillon) [inline static/sass/_pattern_card.scss]: When naming patterns lets try and name them by the purposes. So they can be reused without odd context. I would suggest something like `.p-card--muted` or something.

File: templates/accordion-signup-form.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                        <label class="p-form__label u-no-margin--top" for="canonicalUpdatesOptIn">I would like to receive occasional updates from Canonical by email.</label>&nbsp;<span class="mktFormMsg"></span>
+                      </span>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="col-8">
+
+            </div>
```
Comment (anthonydillon) [inline templates/accordion-signup-form.html]: Remove empty div please

File: templates/accordion-signup-form.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+          <h3 class="p-heading--four">
+            <button class="p-accordion__tab" id="accordion-form-tab" role="tab" aria-controls="#accordion-form" aria-expanded="false">Sign up for email updates</button>
+          </h3>
+        </div>
+      </div>
+      <section class="p-accordion__panel" id="accordion-form" role="tabpanel" aria-hidden="true" aria-labelledby="accordion-form-tab">
+        <form action="https://pages.canonical.com/index.php/leadCapture/save" method="post" id="mktoForm_1212" novalidate="novalidate">
+          <div class="row">
+            <div class="col-12">
+              <p>Choose the topics you're interested in</p>
```
Comment (anthonydillon) [inline templates/accordion-signup-form.html]: Can you encode the apostrophe so it looks a little nicer.

**Comment by spencerbygraves** [comment]
@steverydz thanks, looking good. Can you make some small updates to the spacing please?


**http://insights.ubuntu.com-pr-292.run.demo.haus/**

- [ ] Add 'u-no-padding--top' to the first strip under the featured posts
- [ ] In the sign up box, change "mktFormReq mktField" to margin-top: 1rem;

**http://insights.ubuntu.com-pr-292.run.demo.haus/home**

- [ ] In the open sign up section, can we have margin-top: 1rem; on the rows please
- [ ] Check the chevron size with @anasereijo 

Thanks

**Comment by anasereijo** [comment]
Hi Steve,
It's looking good, thanks!
Just a couple of things...

1. Featured Articles:
can we see with a 3rd card? would be nice to see a webinar

2. Signup card:
seems like some elements have 32px margin can we try 16px so it's the same? 
the amount of copy we see is it fixed? or can we have a bit more to fill up the space?
see bellow:
screen shot 2018-04-04 at 12 39 15

3. Signup accordion:
please amend the margins
can we have animation?
screen shot 2018-04-04 at 12 32 55
when I open the banner I get this selected blue line - not sure if it's my browser or something we can change? or is it part of our guidelines? maybe it's an UX question - FYI @frankiegranato 
screen shot 2018-04-04 at 12 42 23

File: templates/alternate_index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
+{% extends "layout.html" %}
+{% block body %}
+
+<section class="p-strip is-shallow">
+  <div class="row u-vertically-center">
```
Comment (anthonydillon) [inline templates/alternate_index.html]: `u-vertically-center` is not needed here

File: templates/newsletter-form.html [hunk 5]
```diff
@@ -22,7 +22,7 @@ <h3 class="p-card--title p-heading--four">Sign up for email updates</h3>
         </li>
       </ul>
       <div class="mktFormReq mktField">
-          <label class="u-no-margin--top" for="Email" class="mktolabel class="u-no-margin--top">Work email: <span>*</span></label>
```
Comment (anthonydillon) [inline templates/newsletter-form.html]: Nice stop

File: templates/posts.html [hunk 6]
```diff
@@ -37,6 +37,76 @@
...
   <div class="row u-equal-height u-clearfix">
   {% endif %}
+  {% if loop.index0 == 2 %}
+  <div class="col-4 p-card--form">
+    <header class="p-card__header">
+      <h5 class="p-muted-heading">Newsletter signup</h5>
+    </header>
+    <div class="p-card__content">
+      <form action="https://pages.canonical.com/index.php/leadCapture/save" method="post" id="mktoForm_1212" novalidate="novalidate">
+        <h3 class="p-card--title p-heading--four">Select topics you're interested in</h3>
```
Comment (anthonydillon) [inline templates/posts.html]: Can you encode the apostrophe please so its a left hand quotation.

**Comment by anthonydillon** [comment]
The form returns with `newsletter=true` which should show a notification at the top of the page. Please see: https://insights.ubuntu.com/2018/03/21/kernel-team-summary-march-21-2018?newsletter=true

**Comment by steverydz** [comment]
@anasereijo The blue line is a focus state - it's for accessibility

**Comment by steverydz** [comment]
@spencerbygraves The changes you requested are up, except the chevron.
@anasereijo Could you please upload the your chevrons. I used some existing ones here but yours are slightly different I think.

**Comment by anasereijo** [comment]
@steverydz the chevron is a vanilla symbol (height:24px)

**Comment by steverydz** [comment]
@anthonydillon Your requested code changes are up

**Comment by steverydz** [comment]
@anasereijo 

1. There are only two featured articles available right now unfortunately
2. From what I can tell, that is the cut off point for text in WordPress for excerpts. Do you know if that's right @anthonydillon? I can't see a way to change it in the app?
3. I'll fix the margins/spacing. As for animation, this is the vanilla accordion which currently doesn't have an animation. I'd have to change the structure of that component to add it which I think should be part of a larger discussion. I'll raise this in the next dev catch up.

**Comment by anasereijo** [comment]
@steverydz thanks for making those changes.
Let me know when I can see the final version.

Thanks!

**Comment by pmahnke** [comment]
all:

1. the work email is a requirement from marketing
2. with a blog post, you can never be sure of the min-length of an excerpt, so no point in asking for more really.
3. you could just pick another, older article to make sticky, as it will not be shown on the other versions to test
4. looking pretty good I think.

**Comment by steverydz** [comment]
Thanks @pmahnke. I don't have access to the articles on insights to make any of them sticky/featured. I can login to the admin panel but can't do anything once in there.

**Comment by pmahnke** [comment]
@steverydz I have added you to canonical-webmonkeys, which should let you get editor access

**Comment by steverydz** [comment]
Thanks @pmahnke

**Comment by steverydz** [comment]
@anasereijo I've made all of the changes now. You should be able to see them all at the demo link. If it doesn't look right on first load try a hard refresh as your browser may have cached the styles.

**Comment by anasereijo** [comment]
@steverydz 

1. can you please fix that bug we just spoke about on the accordion version?

2. did you made the padding changes on the accordion version? maybe it's me but looks the same

3. please amend the following margins
screen shot 2018-04-05 at 09 51 27

Thank you!

**Comment by spencerbygraves** [comment]
@steverydz on /home the expanded accordion has 24px margin top on the field, can you make it 16px please. Also, there is also a hard space after "I would like to receive...", can we remove that please.

Thanks.

image

**Comment by steverydz** [comment]
@anasereijo @spencerbygraves All changes made

@anasereijo Yes, I've updated the spacing for the accordion and just fixed the accordion bug

**Comment by anasereijo** [comment]
@steverydz 
I still have that bug accordion bug
everything else is looking good thanks

**Comment by steverydz** [comment]
@anasereijo Spacing changes fixed

**Comment by steverydz** [comment]
@anthonydillon I've added the Google Optimize snippet and the GA event. I've given you access to the Optimize account so you can see the setup there. I think I've covered everything but would be good if you could give it a once over.

**Pull request merged**